### PR TITLE
Delete Functionality Rooms and Hotel in MVC

### PR DESCRIPTION
**Pull request by sathish199308**
Delete Functionality Rooms and Hotel in MVC

### CODE REVIEW

File: BookingSystem/APIClient/Interface/IAPIClientHotelRepository.cs [hunk 1]
```diff
@@ -12,5 +12,7 @@ public interface IAPIClientHotelRepository
         Task<List<HotelVM>> GetHotelDetailsAsync();
         Task<HotelVM> GetHotelDetailsByIdAsync(int? HotelID);
         Task<HotelVM> SaveHotelDetailsAsync(ViewHotelData hotel);
+
```
Comment (Backiaraj07) [inline BookingSystem/APIClient/Interface/IAPIClientHotelRepository.cs:15]: use Async in the end of the method name

File: BookingSystem/APIClient/Interface/IAPIClientRoomRepository.cs [hunk 2]
```diff
@@ -12,5 +12,6 @@ public interface IAPIClientRoomRepository
         Task<List<RoomsVM>> GetRoomsDetailsAsync();
         Task<RoomsVM> GetRoomDetailsByID(int? RoomID);
         Task<RoomsVM> SaveRoomsDetailsAsync(ViewRoomsData viewRoomsData);
+        Task<bool> DeleteRoom(int roomId);
```
Comment (Backiaraj07) [inline BookingSystem/APIClient/Interface/IAPIClientRoomRepository.cs:15]: use Async in the end of the method name

File: BookingSystem/Controllers/HotelController.cs [hunk 3]
```diff
@@ -151,5 +151,15 @@ public IActionResult Create(ViewHotelData viewHotelData)
 
 
         }
+
+        [HttpGet]
```
Comment (Backiaraj07) [inline BookingSystem/Controllers/HotelController.cs:155]: use Async in the end of the method name

File: BookingSystem/Controllers/RoomController.cs [hunk 4]
```diff
@@ -45,7 +45,7 @@ public async Task<ActionResult> Create()
         }
 
         [HttpGet]
-        [Breadcrumb("RoomList", FromAction = nameof(Room), FromController = typeof(VenueController))]
+        //[Breadcrumb("RoomList", FromAction = nameof(Room), FromController = typeof(VenueController))]
```
Comment (Backiaraj07) [inline BookingSystem/Controllers/RoomController.cs:48]: Change the controller name in breadcrumb

File: BookingSystem/Views/Hotel/Hotels.cshtml [hunk 5]
```diff
@@ -62,7 +62,8 @@
                 <a asp-action="Create" class="btn btn-primary" asp-route-HotelID="@item.HotelID">
                     <i class="far fa-edit"></i> Edit
                 </a>
-                <a asp-action="Delete" class="btn btn-danger" asp-route-HotelID="@item.HotelID">
+             
+                <a asp-action="DeleteCustomer" class="btn btn-danger" asp-route-HotelID="@item.HotelID">
```
Comment (Backiaraj07) [inline BookingSystem/Views/Hotel/Hotels.cshtml:66]: Give proper action name redirect

**Review by Backiaraj07 — COMMENTED** [review]
Added comments

**Review by Backiaraj07 — APPROVED** [review]
reviewed

**Review by manic1990 — APPROVED** [review]

**Pull request merged**